spec/lib/submission_builder/ty2021/documents/irs1040_spec.rb

```diff
@@ -94,15 +94,15 @@
         expect(xml.at("TaxableIncomeAmt").text).to eq "0"
 
         # Line 28: remaining amount of CTC they are claiming (as determined in flow and listed on 8812 14i
-        expect(xml.at("RefundableCTCOrACTCAmt").text).to eq "0" # TODO: replace this when we calculate this number
+        expect(xml.at("RefundableCTCOrACTCAmt").text).to eq "3600"
```

Comment:
This 3600 number seems arbitrary. Maybe it makes sense to mock `Efile::BenefitsEligibility` in this spec so the 3600 can come from somewhere; same strategy as 1040_pdf_spec. Not a blocker, but I think a very good idea. What do you think?

Reply:
ooo makes sense, added mocks for BenefitsEligibility!